bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2278,7 +2278,12 @@ namespace {
    //- Adding array interface to classes ---------------
    void AddArrayInterface(PyObject *pyclass, PyCFunction func)
    {
+      // Add a getter for the array interface dict to the class.
       Utility::AddToClass(pyclass, "_get__array_interface__", func, METH_NOARGS);
+      // Add the dictionary as property to the class so that it updates automatically if accessed.
+      // NOTE: Since we are not able to add a property easily from C++, we do this in Python.
+      PyObject_CallFunction(PyObject_GetAttrString(gRootModule, "_add__array_interface__"),
```

Comment:
But then, we can add a property to the class by calling the `_add__array_interface__` function in the ROOT module on the class PyObject. I've not found any other way to do this easily.

Reply:
I believe you need to decrement the reference to the return value of `PyObject_CallFunction(PyObject_GetAttrString(gRootModule, "_add__array_interface__")`.

Reply:
Ah you are correct -> https://docs.python.org/2/c-api/object.html

It returns a new reference and does not steal. But does this mean that the "in-place" call of the Python C API is never a good idea? So this leads always to leaking PyObjects?

Reply:
Oh, same thing applys then for `PyObject_CallFunction` since it returns a new reference to the result.

Reply:
Fixed.